docs/docs/cmd/pp/managementapp/managementapp-add.md

```diff
@@ -0,0 +1,50 @@
+# pp managementapp add
+
+Register management application for Power Platform
+
+## Usage
+
+```sh
+m365 pp managementapp add [options]
+```
+
+## Options
+
+`--appId [appId]`
+: Application (client) ID of the Azure AD application registration to register as management app. Specify either `appId`, `objectId` or `name`
+
+`--objectId [objectId]`
+: Object ID of the Azure AD application registration to register as management app. Specify either `appId`, `objectId` or `name`
+
+`--name [name]`
+: Name of the Azure AD application registration to register as management app.. Specify either `appId`, `objectId` or `name`
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+To execute this command the first time you will need to be signed in using the default Azure PowerShell App Registration `1950a258-227b-4e31-a9cf-717495945fc2`. Run `m365 login --appId 1950a258-227b-4e31-a9cf-717495945fc2` before executing this command. If you are using your the default CLI for Microsoft 365 you can use `--appId 31359c7f-bd7e-475c-86db-fdb8c937548e` to register.
```

Comment:
> If you are using your the default CLI for Microsoft 365...

Let's rephrase it because it sounds confusing as it is know. I think that we're trying to say, that after you registered CLI for Microsoft 365 as a management app, you can use it to register additional apps, but then you don't need to specify the ID because that's a default, so perhaps this sentence is unnecessary.

Reading it again, are we trying to say that if you want to register CLI as a management app you should use... Without the command name, it's hard to follow what we mean here 😊

Reply:
Tried to reword, hope it is a bit clearer now.